Governance.md

```diff
@@ -0,0 +1,78 @@
+# Governance of the rules_scala
+
+These bazel rules for scala are under the bazelbuild org but governed by a
+contributors from the community.
+
+## Goals of this document
+
+Clarify:
+0. values of the project.
+1. how to get a PR merged
+2. how long you should need to wait for a review
+3. how to resolve conflicts in project direction
+4. how to make larger and disruptive changes
+
+## Values of rules_scala
+
+1. Correctness: bazel values fully reproducible builds which are reliable and hermetic (only depend on the current state of your repository)
+2. Speed: we want to build as fast as possible. In a large repository, we believe caching and remote parallel builds are the only way to achieve this. Without correctness, we can't cache and use remote builds, so correctness enables speed. We will almost never trade off correctness for speed.
+3. Usability: we want it to be as easy as possible to adopt and to maintain a build with these rules. Usability is a lower priority than correctness or speed. We will virtually never trade either correctness or speed for more usability. Usability includes stability: we seek to absolutely minimize breaking changes. We will prefer to keep some warts or deprecated methods than force users to constantly make changes to keep their builds on the latest version of the rules.
+
+## How to make a PR
+We welcome pull-requests and do all development in the open. Pull requests
+should be in the smallest testable units. Ideally a PR is less than 400 lines of
+change. In rare cases it will be more. If you PR is much larger than that, we
+will likely ask you to split it into smaller changes. You should see a comment
+in 24 business hours on your PR. Your PR *may* be ignored if it has a red CI. It
+is your responsibility to get your PR green, or to ask for clarification if you
+doubt a test failure is due to your change.
+
+Code review increases the quality of our code.  We welcome code review by anyone:
+you don't need to be a maintainer to offer your review.
+
+PRs are merged when 2 or more maintainers have approved the PR and there are no
```

Comment:
2 maintainers for every PR is maybe a high bar?

Reply:
That has been our expectation so far (we sometimes pull the trigger early, but not just merging what stripe wants without taking wix into account has been a commitment). We don't always do it strictly, but if we add more maintainers, I don't think this should be a problem personally.

Reply:
👍 true, the wix thing is a great example of waiting/wanting to flag. Some leeway around minor things seems like a normal course of action and would likely be how it goes (bug fixes)

Reply:
For core feature changes, I'd like to see buy in from at least two maintainers from at least two different orgs/companies.

Reply:
I agree with all that has been written here. I suggest we also decide to have a VC in 2-3 months to discuss this doc and see how we think it’s working out. 
In such a meeting we can decide to lax the rules for “minor” PRs but I wouldn’t do it from the get go.

Reply:
Let's connect on this topic at BazelCon in December. How does that sound?